crates/forge/tests/integration/available_gas.rs

```diff
@@ -8,7 +8,24 @@ fn correct_available_gas() {
     let test = test_utils::test_case!(indoc!(
         r"
             #[test]
-            #[available_gas(11)]
+            #[available_gas(l2_gas: 440000)]
+            fn keccak_cost() {
+                keccak::keccak_u256s_le_inputs(array![1].span());
+            }
+        "
+    ));
+
+    let result = run_test_case(&test, ForgeTrackedResource::CairoSteps);
+
+    assert_passed(&result);
+}
+
+#[test]
+fn correct_available_gas_with_unnamed_argument() {
+    let test = test_utils::test_case!(indoc!(
+        r"
+            #[test]
+            #[available_gas(440000)]
```

Comment:
How did this test work previously with just `11` available gas?

Reply:
it was in some of the threads, it was previously l1, this is (was) l2. As we discussed, I have changed the unnamed to be classic l1, so this test is now back to 11.